config/locales/en.yml

```diff
@@ -3635,3 +3635,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       shipment:
         cannot_ready: "Cannot ready shipment."
       invalid_taxonomy_id: "Invalid taxonomy id."
+    activerecord:
+      models:
+        spree/payment:
```

Comment:
this key is already defined at line 35 so you should append `one` there.

Reply:
on line 35, 'spree/payment' is under 'activerecord.**attributes**', but in my case under 'activerecord.**models**'. Or does it not matter?

Reply:
right. Good catch. I find it a bit odd but if it works :+1: